client/public/locales/es/translation.json

```diff
@@ -77,23 +77,23 @@
   "Desktop": "Escritorio",
   "Layout": "Diseño",
   "Always visible": "Siempre visible",
-  "Save my settings (in the browser store)": "Guardar mis ajustes (en el almacenamiento del navegador)",
-  "<p>Game rules determine if it is allowed to skip levels and if the games runs checks to only allow unlocked tactics and theorems in proofs.</p><1>Note: \"Unlocked\" tactics (or theorems) are determined by two things: The set of minimal tactics needed to solve a level, plus any tactics you unlocked in another level. That means if you unlock <1>simp</1> in a level, you can use it henceforth in any level.</1><p>The options are:</p>": "<p>Las reglas del juego determinan si se permite saltarse niveles y si el juego realiza comprobaciones para permitir únicamente tácticas y teoremas desbloqueados en las pruebas.</p><1>Nota: las tácticas (o teoremas) \"Desbloqueadas\" está determinadas por dos cosas: el conjunto mínimo de tácticas necesarias para resolver un nivel, más cualquier táctica que hayas desbloqueado en otro nivel. Esto significa que si desbloqueas <1>simp</1> en un nivel, puedes usarlo a partir de entonces en cualquier nivel.</1><p>Las opciones son:</p>",
+  "Save my settings (in the browser store)": "Guardar mis ajustes (en el navegador)",
+  "<p>Game rules determine if it is allowed to skip levels and if the games runs checks to only allow unlocked tactics and theorems in proofs.</p><1>Note: \"Unlocked\" tactics (or theorems) are determined by two things: The set of minimal tactics needed to solve a level, plus any tactics you unlocked in another level. That means if you unlock <1>simp</1> in a level, you can use it henceforth in any level.</1><p>The options are:</p>": "<p>Las reglas del juego determinan si puedes saltarte niveles y si puedes usar tácticas y teoremas no aún desbloqueados.</p><1>Nota: las tácticas o teoremas \"Desbloqueados\" están determinados por dos cosas: el conjunto mínimo de tácticas necesarias para resolver un nivel, más cualquier táctica que hayas desbloqueado en otro nivel. Esto significa que si desbloqueas <1>simp</1> en un nivel, puedes usarlo en cualquier otro nivel a partir de entonces.</1><p>Las opciones son:</p>",
   "Game Rules": "Reglas del juego",
   "levels": "niveles",
   "tactics": "tácticas",
-  "regular": "normal",
-  "relaxed": "relajado",
-  "none": "ninguno",
-  "<p>Select a JSON file with the saved game progress to load your progress.</p><1><0>Warning:</0> This will delete your current game progress! Consider <2>downloading your current progress</2> first!</1>": "<p>Seleccione un archivo JSON con el progreso del juego guardado para cargar su progreso.</p><1><0>Advertencia:</0> Esto borrará su progreso actual en el juego. Considere <2>descargar su progreso actual</2> antes</1>",
+  "regular": "normales",
+  "relaxed": "relajadas",
+  "none": "ninguna",
```

Comment:
These adjectives apply to the noun "reglas", so they have to be feminine and plural!